src/Components/CommentAdder.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react"
+import { postComment } from "../Utils/api";
+
+const CommentAdder =({article_id, setComments}) => {
+const [newComment, setNewComment] = useState('')
+const [isLoading, setIsLoading]=useState(false)
+
+const handleSubmit=(event)=>{
+    setIsLoading(true)
+    event.preventDefault();
+    const postBody = {
+        author: "grumpy19",
```

Comment:
hard coded author at this stage is fine, but you may want to make it dynamic later.